api/lttb.md

```diff
@@ -39,28 +39,41 @@ original dataset. The TimescaleDB Toolkit implementation of this takes
 
 |Column|Type|Description|
 |-|-|-|
-|`sortedtimevector`|`SortedTimevector`|A [`timevector`][hyperfunctions-timevectors] object containing the downsampled points. It can be unpacked via `unnest`.|
+|`timevector`|`Timevector`|A [`timevector`][hyperfunctions-timevectors] object containing the downsampled points. It can be unpacked via `unnest`.|
```

Comment:
```suggestion
|`timevector`|`Timevector`|A [`timevector`][hyperfunctions-timevectors] object containing the downsampled points. It can be unpacked with `unnest`.|
```